---
title: "Evolutionary models of Z-linked synthetic suppression gene drives"
author:
- Luke Holman*
- '*\textit{luke.holman@unimelb.edu.au} \vspace{5mm}'
bibliography: references.bib
output:
  pdf_document: default
  html_document: default
  word_document: default
fontsize: 12pt
header-includes: \usepackage{microtype} \usepackage[all]{hypcap} \usepackage{amsmath}
  \usepackage{booktabs} \usepackage{caption} \usepackage[labelfont=bf]{caption} \usepackage{titling}
  \pretitle{\begin{flushleft}\LARGE} \posttitle{\par\end{flushleft}\vskip 0.5em} \preauthor{\begin{flushleft}\large
  \lineskip 0.5em} \postauthor{\par\end{flushleft}} \predate{\begin{flushleft}\large}
  \postdate{\par\end{flushleft}} \usepackage{fancyhdr} \pagestyle{fancy} \fancyhead[LO,LE]{\textsl{\leftmark}}
  \rhead[]{Z-linked gene drives} \usepackage[noabbrev,capitalise]{cleveref} \usepackage{titlefoot}
  \usepackage{amssymb} \usepackage{rotating} \usepackage{lineno} \linenumbers
csl: the-american-naturalist.csl
abstract: My abstract text. My abstract text. My abstract text. My abstract text.
  My abstract text. My abstract text. My abstract text. My abstract text. My abstract
  text. My abstract text. My abstract text. My abstract text. My abstract text. My
  Keyword1, Keyword2, Keyword3, Keyword4, Keyword5. abstract text. My abstract text.
  My abstract text. \vspace{5mm}\par\noindent\textbf{Keywords:}
---

```{r setup, include=FALSE}
knitr::opts_chunk$set(echo = FALSE)
# pdf(file = "../figures/example_fig_123456.pdf"); plot((-10:10)^2); dev.off()
```

\maketitle\unmarkedfntext{*School of BioSciences, The University of Melbourne, Victoria, Australia.}

\newpage
# Introduction
Here is a reference [e.g. @Holman_2018], and here's a link to a figure (\autoref{example_plot}).

Aims:
  - Compare low and high cost shredders ability to suppress the pop. High cost spreads worse since Z*W females have fewer offspring, but it also means fewer offspring are born!
  - Local vs global release?
  - butterfly vs worm vs bird
  - effect of m/f dispersal and patch structure
  - is the Z conversion needed for spread?
  - effect of male weighting and softness of selection on females

# Methods

## Overview
We model a finite population of dioecious diploids with ZW sex determination, living in a landscape containing $k$ discrete habitat patches. The model considers the demography and evolution of the population following the release of a number of males carrying a Z-linked allele that is capable of gene drive in females (e.g. through W-shredding) and/or in males (e.g. via gene consersion). Our principle aim is to identify the key factors that determine whether the Z-linked genetic element (hereafter Z\*) causes extinction of the population. The model is a stochastic individual-based simulation written in the R scripting language, and was run on the `Spartan` High Performance Computing system at the University of Melbourne. An accompanying website presents the annotated code used to run the model and generate all the figures (link). 



## Loci and alleles
Each male individual in the simulation carries a Z-linked locus and two autosomal loci, each with two alleles. Each female carries a single allele at the Z-linked locus plus a single W-linked allele, as well as two alleles at both of the autosomal loci.

There are three possible Z-linked alleles: a wild-type Z allele (denoted Z+) which is vulnerable to gene drive; a gene drive allele (denoted Z\*) which can perform gene drive in females and/or males, and a resistant allele (Zr) which is immune to gene conversion. Similarly, there are two possible W-linked alleles: a wild-type W chromosome (W+) that is vulnerable to gene drive by the Z\* allele, and a resistant W chromosome (Wr) that is immune to gene drive.

The two autosomal loci are denoted A/a and B/b, and confer resistance to W-shredding and gene conversion respectively. One might call this 'trans-acting resistance', since the resistance is not mediated by a homolog of the gene drive allele, as opposed to the 'cis-acting resistance' conferred by the Zr and Wr alleles. Specifically, we assume that the A/a locus carries alleles a and A, where the A allele is dominant and confers immunity to gene drive (e.g. W-shredding) in females. The B/b autosomal locus carries alleles b and B, where B is dominant and confers immunity to gene drive (e.g. gene conversion) in males. 

## Production of gametes by meiosis
We assume that the A/a and B/b loci segregate independently during meiosis, and they display standard Mendelian inheritance. Inheritance of the sex chromosomes is also Mendelian, except for certain genotypes that carry a single copy of the Z\* gene drive allele. Provided they do no carry a resistant Wr chromosome or at least one A allele, 

Firstly, females with the genotypes Z\*W+aaBB, Z\*W+aaBb, or Z\*W+aabb produce a fraction $\frac{1}{2}(1 + p_{shred})$ of Z-bearing gametes and $\frac{1}{2}(1 - p_{shred})$ W-bearing gametes. Therefore, these three female genotypes produce more than 50% male offspring if $p_{shred} > 0$, due to the shortage of W chromosomes in their gametes. Note that the gamete frequencies produced by Z\*Wr females, or by Z\*W+ females carrying at least one A allele, conform to the standard Mendelian expectations.

Secondly, males with the genotypes Z\*Z+AAbb, Z\*Z+Aabb, or Z\*Z+aabb males produce a fraction $\frac{1}{2}(1 + p_{conv})$ of gametes that carry the Z\* allele and $\frac{1}{2}(1 - p_{conv})$ gametes that carry the Z+ allele. Thus, $p_{conv} > 0$ indicates that Z\* is over-represented in the gametes of these three male genotypes. Note that the gamete frequencies produced by Z\*Zr males, or by Z\*Z+ males carrying at least one B allele, conform to the standard Mendelian expectations.


## Calculating female fecundity
To begin the breeding phase of the lifecycle, we first determine the number of offspring produced by each female in the meta-population. The expected fecundity of each female is determined by three factors: her genotype, the density of males and females in the local patch and/or the meta-population, and global parameters in the model. 

Specifically, the expected fecundity of female $i$ ($F_i$) is calculated as

$$F_i = (1 + w_i r(1 - (D_i / K) ^ c))$$

where $w_i$ is the relative fitness of female $i$ (possible range: 0 to 1, where 1 is the fitness of the wild type Z+W+aabb females), $D_i$ is the 'density' experienced by female $i$, $K$ is the carrying capacity, and $r$ and $c$ are constants that scale the maximum possible fecundity and the shape of density-dependence, respectively. 

To ensure that the simulation captures variation in life history and ecology between species (see Introduction), we calculate the density $D_i$ in various ways across different simulation runs. First, we define the 'global density' $d_g$, which is experienced equally by every female in every patch, as 

$$d_g = \sum_{i=1}^{N_f} w_i + \delta N_m$$	

where $N_f$ and $N_m$ is the number of females and males across all patches, the first term is the sum of the fitnesses of all these females, and $\delta$ is a constant (range: $0-\infty$) that scales the effect of each male on $d_g$, relative to a female with fitness $w_i = 1$. This formulation means that females with high relative fitness (i.e. fecundity) have a stronger effect on the global density than do low-fitness females. We also assume that each male contributes a fixed amount to the global density, irrespective of his genotype/fitness (male fitness is only used to determine male mating success; see below). The parameter $\delta$ represents sex differences in ecological niche use and behaviours that affect female fecundity. For example, we might expect $\delta<1$ in species where males and females utilise very different environmental niches, or $\delta>1$ in species with strong inter-locus sexual conflict. 

Second, we define the 'local density' $d_l$, which is experienced by every female in a particular patch, as 

$$d_l = \sum_{i=1}^{n_f} w_i + \delta n_m$$	

where $n_f$ and $n_m$ are the numbers of females and males in the patch. As before, this formulation means that $d_l$ depends on the fitness of the females in the patch, but only on the number of males (multiplied by the constant $\delta$).

Finally, the overall density $D_i$ is a composite of the global and local densities given by $D_i = \psi d_g + (1 - \psi)d_l$. The parameter $\psi$ scales the importance of global and local density to female fecundity. When $\psi = 0$, only local density matters and selection on females is entirely "soft", while when $\psi = 1$ only global density matters and selection on females is completely "hard" (REFERENCE). Intermediate values of $\psi$ produce a mixture of hard and soft selection on females, and the growth rate of population depends on density at both scales.

Once we have calculated the expected fecundity of each female ($F_i$), we generate the realised fecundity of the female by randomly sampling from a Poisson distribution with $\lambda = F_i$ (allowing for stochastic as well as deterministic variation in fecundity between females). If the resulting number of offspring exceeds the global carrying capacity $K$, we randomly cull the offspring until $K$ are left. 


## Competition between males

Next, we select fathers for each of the offspring. We assume that all breeding occurs within patches, such that males are only in competition with males from the same patch (i.e. selection on males is always "soft"; REFERENCE). If the patch contains $k$ different male genotypes and $n_1, n_2, ... n_k$ males of each genotype, the probability that a male of genotype $k$ is the father of any given offspring is 

$$p_j = \frac{n_{k}w_k}{\sum_{i=1}^{k}n_{i}w_i}$$

such that relatively common and/or high-fitness male genotypes are more likely to sire offspring. This formulation means that we assume that both sexes potentially reproduce with multiple different partners.

Finally, we randomly generate a genotype for each resulting offspring, based on the genotypes of its parents and the associated expected zygote frequencies. Offspring are born in the same patch as their parents, and the parental generation is removed from the population after reproducing (i.e. we assume discrete, non-overlapping generations). Female and male offspring disperse to another patch with probabilities $x_f$ and $x_m$ respectively. We model two types of dispersal, in separate simulations: local dispersal, in which offspring move to one of the two neighbouring patches with equal probability (recalling that the $k$ patches are arranged linearly in a ring), or global dispersal, in which dispersing offspring can land in any of the other patches.

## Running the simulation
We first initialise the population, with specified (typically low or zero) frequencies for the Zr, Wr, A and B alleles. We then iterated the population for 100 generations of burn-in, to allow the population to reach its equilibrium size and genotypic composition. We then introduce $m_{release}$ males with the genotype Z\*Z\*aabb, representing the release into the wild of a laboratory-reared strain homozygous for the driving Z and for autosomal factors conferring susceptibility to drive. In some simulations, all the Z\*Z\*aabb males were released in a single patch, while in others the $m_{release}$ males were randomly and evenly divided across all $k$ patches. We continued to cycle through the lifecycle (birth, migration, breeding, death) until either A) the driving Z\* allele went extinct, B) the population went extinct, C) the Wr chromosome went to fixation (making population suppression impossible), D) the Z\* allele fixed, but failed to cause population extinction, or E) 400 generations had elapsed. We recorded which of these five outcomes occurred, as well as the allele frequencies, population size, and sex ratio at each generation. 



# Results

  - Note that when females hardly migrate, the Wr is slow to spread across patches. It only has a good invasion probability if Z* is present, otherwise it's neutral or costly

<!-- ![My figure legend. My figure legend. My figure legend. My figure legend. My figure legend. My figure legend. My figure legend. \label{example_plot}](../figures/example_fig_123456.pdf) -->

# Discussion


# Acknowledgements
So long, and thanks for all the fish!

# References
<div id="refs"></div>

\newpage
# Supporting information
