---
title: "Evolutionary models of synthetic \\textit{Z}-linked suppression gene drives"
author:
- Luke Holman*
- '*\textit{luke.holman@unimelb.edu.au} \vspace{5mm}'
bibliography: references.bib
output:
  pdf_document: default
  html_document: default
  word_document: default
fontsize: 12pt
header-includes: \usepackage{microtype} \usepackage{amsmath}
  \usepackage{booktabs} \usepackage{caption} \usepackage[labelfont=bf]{caption} \usepackage{titling}
  \pretitle{\begin{flushleft}\LARGE} \posttitle{\par\end{flushleft}\vskip 0.5em} \preauthor{\begin{flushleft}\large
  \lineskip 0.5em} \postauthor{\par\end{flushleft}} \predate{\begin{flushleft}\large}
  \postdate{\par\end{flushleft}} \usepackage{fancyhdr} \pagestyle{fancy} \fancyhead[LO,LE]{\textsl{\leftmark}}
  \rhead[]{Z-linked gene drives} \usepackage[noabbrev,capitalise]{cleveref} \usepackage{titlefoot}
  \usepackage{amssymb} \usepackage{rotating} \usepackage{lineno} \linenumbers
csl: phil-trans.csl
abstract: My abstract text. 
---

```{r setup, include=FALSE}
knitr::opts_chunk$set(echo = FALSE)
source("../code/table_1_code.r")
```

\maketitle\unmarkedfntext{*School of BioSciences, The University of Melbourne, Victoria, Australia.}

\textbf{Keywords: \textit{W}- and \textit{X}-shredders, Lepidopteran pests, meiotic drive, population control, schistosomiasis, selfish genes}.

<!-- \usepackage[all]{hypcap} -->

<!-- Here is a reference [e.g. @Holman_2018], and here's a link to a figure (\autoref{example_plot}). -->


\newpage
# Introduction
It is becoming increasingly technically feasible to genetically engineer natural populations in order to control pathogens, pathogen vectors, agricultural pests, and invasive species using 'gene drives' [@gantz2015hi; @hammond2016cr; @wang2016cr; @prowse2017do; @kyrou2018cr; @noble2018cu]. Gene drives assist the propagation of engineered genes through populations using a range of mechanisms including gene conversion, poison-antidote systems, segregation distortion, and genetic incompatibility [@lindholm2016ec; @champer2016ch]. For example, the CRISPR-Cas9 gene editing system allows one to produce a transgenic insertion that is transmitted to almost 100% of the offspring of heterozygous individuals instead of the usual 50% (due to targeted cutting of the homologous chromosome followed by repair using the gene drive as a template). CRISPR-Cas9 can be used to create 'replacement drives', which propagate a human-beneficial allele that would not otherwise spread, e.g. an allele that interferes with the tranmission of malaria by mosquitoes [@gantz2015hi; @marshall2015gene]. CRISPR-Cas9 can also be used to create suppression drives, which aim to cause extinction (or at least reduce the size) of the target population by introducing genes with lethal or sterilising effects (MORE HERE [@hammond2016cr; @kyrou2018cr]), or by skewing the population sex ratio -- typically towards males MORE HERE [@beaghton2017ve; @burt2018se; @papathanos2018re].

Many recent theoretical papers have investigated the feasibility and possible consequencies of emerging gene drive technologies. For example, Noble et al. [-@noble2018cu] used models to show that the most basic implementation of a CRISPR gene drive might be highly invasive and could rapidly spread to fixation across whole meta-populations, which may often be undesirable. Conversely, other models [@drury2017cr; @unckless2017ev] have concluded that 'resistant' alleles that are immune to being cut and replaced by CRISPR gene drives could prevent them from spreading and acheiving their aims. The issue of resistance is compounded because the standard implementation of CRISPR drives (but see [@unckless2017ev; @prowse2017do; @kyrou2018cr]) tends to create its own resistance alleles as a by-product, through a process called non-homologous end joining (NHEJ; [@gantz2015mu; @gantz2015hi; hammond2016cr; wang2016cr; @unckless2017ev]). Given the potential safety, ethical, and socio-political concerns surrounding gene drives, several theoretical papers have focused on the potential to create gene drives that would go extinct once their job is done [@min2017da; @burt2018se], would stay confined to particular populations [@maselko2018ge], and/or could be reversed once they have spread [@vella2017ev]. 

Here, I focus on the evolutionary dynamics of a _Z_-linked suppression gene drive. The simulation is inspired by proposals for various types of CRISPR-Cas9 _Z_-linked gene drives that have been proposed by Kevin Esvelt and colleagues (see [www.sculptingevolution.org/genedrives/current/schistosomiasis](); at the time of writing, these ideas have not been published in a journal or pre-print archive). Various _Z_-linked suppression drives proposed by Esvelt and colleagues are shown schematically in Figure 1. The gene drive would enjoy a transmission advantage in _ZW_ females, and optionally also in _ZZ_ males. Esvelt et al. propose that _Z_-linked drives could be used to control the trematode parasites (_Schistosoma_ spp.) responsible for the deadly disease schistosomiasis, though _Z_-linked drives could theoretically be used to control any organism with female-heterogametic sex determination (such as Lepidopteran pests or invasive birds).

A _Z_-linked gene drive could suppress populations by biasing gametogenesis in females, for example by cutting unique sequences on the _W_ chromosome in order to destroy it; such a gene drive would be a '_W_-shredder', similar to the _X_-shredders that are being developed to control _XY_ species [@north2013mo; @galizi2014sy; @papathanos2018re]. Females carrying the gene drive would thus produce relatively few _W_-bearing eggs, and therefore produce mainly drive-carrying sons. Esvelt et al. point out that the total number of offspring produced by these drive-carrying females will depend on multiple factors, such as the timing of the drive mechanism, the strength of individual-level fitness costs to drive carriers, and the ecology of the target species, with attendant consequences for the evolutionary dynamics of the drive. For example, for some _W_-shredders, drive females might produce roughly the same number of offspring as a wild-type female (only with a preponderance of sons) because the _W_ chromosome is destroyed at an early enough stage in oogenesis that the number of mature ggs is not affected (Figure 1, 1A). Alternatively, drive-carrying females might produce half (or less than half) the number of offspring, e.g. if the drive works by destroying all ova or offspring that inherit a _W_ chromosome, and females cannot compensate by producing more. As an alternative to _W_-shredders, Esvelt et al. also proposed that one could suppress populations using a _Z_-linked locus that caused sterility or lethality in females. If this female-harming gene was capable of gene drive in males (see below), it could perhaps reach high enough frequencies to suppress the population.

Esvelt and colleagues also note that if the _W_-shredder or female-harming _Z_-linked locus caused gene drive in males, it would probably spread through the population faster and be more likely to result in extinction. Male gene drive could be accomplished using 'standard' CRISPR-Cas9 gene conversion (REF), whereby the driving _Z_ locus would convert the wild type locus using homing endonuclease activity followed by DNA repair inside male heterozygotes, leading these males to produce mostly drive-carrying sperm and offspring. 

Impressed by the promise of these ideas, I wrote an evolutionary simulation that can accommodate all of these hypothesised types of _Z_-linked drives. We aimed to test which properties of the gene drive and the ecology of the target species are critical to determining the likelihood and speed with which the gene drive causes extinction. For example, the gene drive will presumably spread faster if it can bias transmission in both sexes, but perhaps a female-only gene drive (which might sometimes be easier to engineer) would be perfectly adequate. Also, since the population will become more male-biased as the gene drive invades, there will be eco-evo feedback (REF) that might affect the evolutionary outcome in non-intuitive ways. For example, the altered sex ratio might intensify the fitness advantage accruing to any resistant _W_ chromosomes or autosomal modifiers that prevent _W_-shredding (due to Fisherian selection for an even sex ratio; [@holman2015co]), relative to that observed in models dealing with resistance to gene drives carried on autosomes [@unckless2017ev; @drury2017cr]. Moreover, the change in sex ratio could affect the demographics of the population, particularly if males and females contribute differentially to density-dependent population growth [@rankin2007ma], or have different dispersal rates [@li2019sex]. The model incorporates the possibility that _Z_-linked resistance alleles are sometimes formed through NHEJ in males that are heterozygous for the drive allele [@gantz2015mu; @gantz2015hi; hammond2016cr; wang2016cr; @unckless2017ev]. It is not clear _a priori_ whether the creation of resistant _Z_-linked alleles by NHEJ is a equally problematic for a _Z_-linked gene drive as it is for an autosomal drive, because it would only hinder gene conversion in males, assuming that NHEJ does not occur in response to _W_-shredding (which is likely, because the _W_-shredder could be designed to target many repetitive regions of the _W_ chromosome).


**Table 1**: List of variables, and their corresponding parameter(s) in the model, which were varied in order to study their effects on the likelihood of population extinction.
```{r xtable, results="asis", fig.cap="Zfgzf"}
print_table1()
```



# Methods

## Overview

We model a finite population of dioecious diploids with _ZW_ sex determination, living in $j$ discrete habitat patches which are arranged linearly in a ring (preventing 'edge effects'). The model considers the demography and evolution of a population into which $n_{release}$ males carrying the _Z_-linked gene drive are released. The drive allele is capable of gene drive in females (e.g. through _W_-shredding) and optionally also gene drive in males (e.g. via gene consersion). The life cycle consists of discrete generations, each composed of the following phases: birth, dispersal between patches, breeding with patches, and death of the parental generation. The species has 3 loci with 2 or 3 alleles each, some of which potentially show non-Mendelian inheritance. The equilibrium population size was roughly 10,000 in all simulations prior to the release of the gene drive. The model is a stochastic individual-based simulation written in R 3.4.0 (REFERENCE) and was run on `Spartan`, a computer cluster at the University of Melbourne. An accompanying website presents the R scripts (with annotations) used to run the model and generate the figures (link). 


## Loci and alleles

Each male individual in the simulation carries one _Z_-linked locus and two autosomal loci, each with two alleles. Each female carries a single allele at the _Z_-linked locus plus a _W_ chromosome, as well as two alleles at both of the autosomal loci.

There are three possible _Z_-linked alleles: a wild-type allele (denoted _Z+_) which is vulnerable to gene drive; a gene drive allele (_Z\*_), and a resistant allele (_Zr_) which is immune to gene drive. Similarly, there are two possible types of _W_ chromosomes: a wild-type _W_ chromosome (_W+_) that is vulnerable to gene drive by the _Z\*_ allele, and a resistant _W_ chromosome (_Wr_) that is immune to gene drive.

The two autosomal loci are denoted _A/a_ and _B/b_, and control immunity to _W_-shredding and gene conversion respectively. _A/a_ and _B/b_ could be called 'trans-acting' resistance loci, since they are at a different locus (indeed, a different chromosome) to the gene drive allele, in contrast to the 'cis-acting' resistance conferred by the _Zr_ and _Wr_ alleles. The _A_ allele is dominant to _a_ and confers immunity to _Z_-linked gene drive (e.g. _W_-shredding) in females. The _B_ allele is dominant to _b_ and confers immunity to _Z_-linked gene drive (e.g. gene conversion) in males. 


## Calculating female and male fitness

We assume that wild-type individuals (i.e. those lacking any drive or resistance) have a fitness ($w$) of 1, while other genotypes have $0 \le w \le 1$. The fecundity of females carrying the gene drive is reduced by a factor $1 - C_f$, where a small $C_f$ implies minimal costs (e.g. because the _Z_-shredding occurs early enough that lost ova/offspring can be replaced), $C_f = 0.5$ could represent the case where all daughters die and are not replaced, and $C_f = 1$ means that gene drive females are completely sterile (which is useful for modelling a female-sterilising _Z_-linked suppression drive as opposed to a _W_-shredder). Similarly, the fitness of males carrying the gene drive is reduced by a factor $1 - C_m$; male fitness is used in the calculation of mating success (see below). Furthermore, the resistant chromosomes _Wr_ and _Zr_ are assumed to reduce fitness by a factor $1 - C_w$ and $1 - C_z$ respectively. All costs are multiplicative; for example, a _Z\*Zr_ male would have fitness $(1 - C_m)(1 - C_z)$. Additionally, all costs are assumed to be dominant, meaning that having one drive or resistance allele is equally costly as having two.

AUTOSOMES?


## Gamete production and gene drive

We assume that the _A/a_ and _B/b_ loci segregate independently during meiosis and display standard Mendelian inheritance. Inheritance of the sex chromosomes is also Mendelian, except for certain genotypes carrying one copy of _Z\*_. 

Firstly, _Z\*W+aaBB_, _Z\*W+aaBb_, and _Z\*W+aabb_ females produce a fraction $\frac{1}{2}(1 + p_{shred})$ of _Z_-bearing gametes and $\frac{1}{2}(1 - p_{shred})$ _W_-bearing gametes. Therefore, these three female genotypes produce >50% sons when $p_{shred} > 0$, due to the shortage of _W_ chromosomes in their gametes. Note that the gamete frequencies of _Z\*Wr females_, or of females carrying at least one _A_ allele, conform to the standard Mendelian expectations due to resistance.

Secondly, _Z\*Z+AAbb_, _Z\*Z+Aabb_, and _Z\*Z+aabb_ males produce a fraction $\frac{1}{2}(1 + p_{conv} - p_{conv} p_{nhej})$ of gametes carrying the _Z\*_ allele, $\frac{1}{2}(1 - p_{conv})$ gametes carrying the _Z+_ allele, and $\frac{1}{2}(p_{conv} p_{nhej}))$ gametes carrying the _Zr_ allele. Thus, gene conversion occurs in males if $p_{conv} > 0$, meaning that the _Z\*_ allele is over-represented in the gametes of these three male genotypes. The parameter $p_{nhej}$ represents non-homologous end joining, in which an endonuclease-based gene drive fails to copy itself to the homologous chromosome, and instead deletes its target site and thereby creates a resistant allele. The gamete frequencies of _Z\*Zr_ males, or of males carrying at least one _B_ allele, conform to the standard Mendelian expectations due to resistance.


## Calculating female fecundity
In the breeding phase of the lifecycle, we first determine the number of offspring produced by each female in the population. We first calculate the expected fecundity of each female, which is affected by three factors: the female's genotype, the density of males and females in the local patch and/or in the full population, and some global parameters in the model. 

Specifically, the expected fecundity of female $i$ ($F_i$) is calculated as

$$F_i = (1 + w_i r(1 - (D_i / K) ^ \alpha))$$

where $w_i$ is the relative fitness of female $i$ (possible range: 0 to 1, where 1 is the fitness of the wild type _Z+W+aabb_ females), $D_i$ is the 'density' experienced by female $i$, $K$ is the carrying capacity, and $r$ and $c$ are constants that scale the maximum possible fecundity and the shape of density-dependence, respectively. Thus, we assume that offspring production is density-dependent according to a Richards model [@fowler1981de].

To ensure that the simulation captures various possible types of life history and ecology (see Introduction), we calculate the density $D_i$ in various ways across different simulation runs. First, we define the 'global density' $d_g$, which is experienced equally by every female in every patch, as 

$$d_g = \sum_{i=1}^{N_f} w_i + \delta N_m$$	

where $N_f$ and $N_m$ is the number of females and males across all patches, the first term is the sum of the fitnesses of all these females, and $\delta$ is a constant (range: $0-\infty$) that scales the effect of each male on $d_g$, relative to a female with fitness $w_i = 1$. This formulation means that females with high relative fitness (i.e. fecundity) have a stronger effect on the global density than do low-fitness females. We also assume that each male contributes a fixed amount to the global density, irrespective of his genotype/fitness (male fitness is only used to determine male mating success; see below). The parameter $\delta$ represents sex differences in ecological niche use and behaviours that affect female fecundity. For example, we might expect $\delta<1$ in species where males and females utilise very different environmental niches, or $\delta>1$ in species with strong inter-locus sexual conflict. 

Second, we define the 'local density' $d_j$, which is experienced by every female in patch $j$, as 

$$d_j = \sum_{i=1}^{n_{f,j}} w_i + \delta n_{m,j}$$	

where $n_{f,j}$ and $n_{m,j}$ are the numbers of females and males in patch $j$. As before, this formulation means that $d_j$ depends on the fitnesses of the females in the patch, as well as the number of males (scaled by the constant $\delta$).

Finally, the overall density experienced by female $i$ in patch $j$ ($D_i$) is a composite of the global and local densities given by $D_i = \psi d_g + (1 - \psi)d_j$. The parameter $\psi$ scales the importance of global and local density to female fecundity. When $\psi = 0$, only local density matters and selection on females is entirely "soft", while when $\psi = 1$ only global density matters and selection on females is completely "hard" (REFERENCE). Intermediate values of $\psi$ produce a mixture of hard and soft selection on females, and the growth rate of population depends on density at both scales.

Once we have calculated the expected fecundity of each female ($F_i$), we generate the realised fecundity of the female by randomly sampling from a Poisson distribution with $\lambda = F_i$ (allowing for stochastic variation in fecundity between females with equal $F_i$). If the resulting number of offspring exceeds the global carrying capacity $K$, we randomly cull the offspring until $K$ are left. 


## Competition between males

After determining how many offspring each female produces, we determine the fathers of each of these offspring. We assume that all breeding occurs within patches, such that males only compete for mating/fertilisation with males from the same patch (i.e. selection on males is always "soft"; REFERENCE). If the patch contains $k$ different male genotypes and there are $n_1, n_2, ... n_k$ males of each genotype, the probability that a male of genotype $k$ is the father of any given offspring is 

$$p_j = \frac{n_{k}w_k}{\sum_{i=1}^{k}n_{i}w_i}$$

such that relatively common and/or high-fitness male genotypes are more likely to sire offspring. This formulation means that we assume that both sexes potentially reproduce with multiple different partners.

## Reproduction, mutation and dispersal

After picking the parents, we randomly generate each offspring's genotype based on the gamete (and thus zygote) frequencies that are expected from the parental genotypes. Offspring are born in the same patch as their parents, and the parental generation is replaced by the offspring generation (i.e. we assume discrete, non-overlapping generations). 

When an offspring is created, each _Z+_ allele it carries has a chance $\mu_Z$ to mutate to a _Zr_ allele, and _vice versa_ (i.e. mutation in both directions is equally probable). Similarly, each _W+_ allele has a chance $\mu_W$ to mutate to a _Wr_ allele, and _vice versa_. 

Female and male offspring disperse to another patch with probabilities $x_f$ and $x_m$ respectively. We model two types of dispersal, in separate simulations: local dispersal, in which offspring move to one of the two neighbouring patches with equal probability (recalling that the patches are arranged linearly in a ring), or global dispersal, in which dispersing offspring can land in any of the other patches.


## One compete run of the simulation

We first initialised a population of 10,000 individuals (i.e. the carrying capacity, $K$) with low or zero frequencies of _Zr_, _Wr_, _A_ and _B_ alleles, higher frequencies of the wild type _Z+_, _W+_, _a_, and _b_ alleles, and zero _Z\*_ gene drive alleles. We then iterated the population for 50 generations of burn-in, to allow the population to approach demographic and genotypic equilibrium. We then introduced $n_{release}$ males with the genotype _Z\*Z\*aabb_, which represents the release into the wild of a laboratory-reared strain homozygous for the driving _Z_ and for autosomal factors conferring susceptibility to drive. Males are released after density-dependent regulation of female fecundity, but before picking fathers for the offspring. In some simulations, all the _Z\*Z\*aabb_ males were released in a single patch, while in others the $n_{release}$ males were randomly and evenly divided across all $k$ patches. We continued to cycle through the lifecycle (birth, migration, breeding, death) until either A) the driving _Z\*_ allele went extinct, B) the population went extinct, C) the _Wr_ chromosome went to fixation (making population suppression impossible), D) the _Z\*_ allele fixed, but failed to cause population extinction, or E) 900 generations had elapsed. We recorded which of these five outcomes occurred, as well as the allele frequencies, population size, and sex ratio at each generation. 

## Investigating the parameter space
For each of the parameters in Table 1, we selected two or more possible parameter values (e.g. high versus low rates of _W_-shredding $p_{shred}$; many versus few patches $k$). We then ran the model once for all possible combinations of these parameter values (n = XXX model runs). The aim was to measure the 'main effect' of each parameter across a background of assumptions for the other parameters, as well as to investigate all the possible 2-way interactions between the parameters (e.g. to test if the effect of $p_{shred}$ depends on $k$ and _vice versa_). 


# Results

## Three illustrative simulation runs
Blah blah

## Main effects of each parameter
Figure 2 shows the main effects of each of the parameters in the model, arranged in order of importance to the evolutionary outcome. 

  - Note that when females hardly migrate, the Wr is slow to spread across patches. It only has a good invasion probability if Z* is present, otherwise it's neutral or costly

<!-- ![My figure legend. My figure legend. My figure legend. My figure legend. My figure legend. My figure legend. My figure legend. \label{example_plot}](../figures/example_fig_123456.pdf) -->

# Discussion
- schistos have large males, small females
- birds often have sex-biased dispersal
- females may be more demographically limiting leps, since it is them that lays the eggs


# Acknowledgements
So long, and thanks for all the fish!

# References
<div id="refs"></div>

\newpage
# Supporting information
