---
title: "Supplementary Material"
subtitle: "From: Holman 2019, _Evolutionary simulations of _Z_-linked suppression gene drives_"
output: pdf_document
header-includes: 
  \usepackage{booktabs}
  \usepackage{microtype}
  \usepackage{longtable}
  \usepackage[margin=1in]{geometry}
---


```{r setup, include=FALSE}
knitr::opts_chunk$set(echo = FALSE, warning = FALSE, message = FALSE)
library(kableExtra)
library(stringr)
library(grid)

make_table <- function(file, LT = TRUE, digits = getOption("digits"), math_mode = FALSE){
  tabl <- readRDS(file)
  tabl[,2] <- prettyNum(tabl[,2], big.mark = ",", scientific = FALSE)
  
  if(!math_mode){
    kable(tabl, "latex", longtable = LT, booktabs = TRUE, digits = digits) %>%
      kable_styling(latex_options = c("repeat_header", "scale_down"), font_size = 7)
  } else {
    kable(tabl, "latex", longtable = LT, booktabs = TRUE, digits = digits, escape = FALSE, table.envir = "array*") %>%
      kable_styling(latex_options = c("repeat_header", "scale_down"), font_size = 7)
  }
}

n_model_runs <- unname(readRDS("../data/number_of_runs.rds")[1,])
```

The R scripts used to run the model and generate all figures and tables can be viewed at [https://lukeholman.github.io/W_shredder/](), along with annotations explaining the code.

## Supplementary Methods

### Calculating fitness

Individuals with no _Z\*_ alleles have an intrinsic fitness of $w$ = 1, while other genotypes have $0 \le w \le 1$. The fecundity of females carrying _Z\*_ is reduced by a factor $1 - c_f$. Small $c_f$ implies minimal costs (e.g. because mothers replace lost gametes/offspring and/or sib-sib competition is intense), $c_f = 0.5$ could represent the case where all daughters die and are not replaced, and $c_f = 1$ means that females carrying _Z\*_ are completely sterile. Setting $c_f = 1$ allows simulation of a female-sterilising _Z_-linked drive. Similarly, the fitness of males carrying _Z\*_ is reduced by a factor $1 - c_m$; male fitness determines mating success (see below). For simplicity, I assume that the resistance alleles $W^r$, $Z^r$, _A_ and _B_ are cost-free. Also, the costs of _Z\*_ to males were assumed to be dominant, such that _Z\*_$Z^+$ males and _Z\*Z\*_ males had equal fitness.


### Gamete production and gene drive

I assume that the _A/a_ and _B/b_ loci segregate independently during meiosis and display standard Mendelian inheritance. Inheritance of the sex chromosomes is also Mendelian except for certain genotypes carrying one _Z\*_ allele. 

Firstly, _Z\*_$W^+$_aaBB_, _Z\*_$W^+$_aaBb_, and _Z\*_$W^+$_aabb_ females produce a fraction $\frac{1}{2}(1 + p_{shred})$ of _Z_-bearing gametes and $\frac{1}{2}(1 - p_{shred})$ _W_-bearing gametes. Therefore, these three female genotypes produce >50% sons when $p_{shred} > 0$, due to the shortage of _W_ chromosomes in their gametes. The gamete frequencies of _Z\*_$W^r$ females, or of females carrying at least one _A_ allele, conform to the standard Mendelian expectations due to resistance.

Secondly, _Z\*_$Z^+$_AAbb_, _Z\*_$Z^+$_Aabb_, and _Z\*_$Z^+$_aabb_ males produce a fraction $\frac{1}{2}(1 + p_{conv} - p_{conv} p_{nhej})$ of gametes carrying the _Z\*_ allele, $\frac{1}{2}(1 - p_{conv})$ gametes carrying the $Z^+$ allele, and $\frac{1}{2}(p_{conv} p_{nhej}))$ gametes carrying the $Z^r$ allele. Thus, gene conversion occurs in males if $p_{conv} > 0$, meaning that the _Z\*_ allele is over-represented in the gametes of these three male genotypes. The parameter $p_{nhej}$ represents the creation of resistance alleles via non-homologous end joining, in which the gene drive fails to copy itself to the homologous chromosome, and instead induces an indel mutation that creates a resistant allele. The gamete frequencies of _Z\*_$Z^r$ males, or of males carrying at least one _B_ allele, conform to the standard Mendelian expectations due to resistance.


### Calculating female fecundity
In the breeding phase of the lifecycle, the simulation first determines the number of offspring produced by each female. The expected fecundity of female $i$ ($F_i$) is affected by three factors: the female's genotype, the density of males and females in the local patch and/or in the full population, and some global parameters in the model, as follows:

\begin{equation}
F_i = (1 + w_i r(1 - (D_i / K) ^ \alpha))
\end{equation}

where $D_i$ is the 'density' experienced by female $i$, $w_i$ is her fitness, $K$ is the carrying capacity, and $r$ and $\alpha$ are constants that control the maximum possible fecundity and the shape of density-dependence respectively (function from [@fowler1981de]).

To ensure that the simulation captures various possible types of life history and ecology, I calculated density $D_i$ in various ways in different simulation runs. First, I define the global density $d_g$, which acts equally on every female in every patch, as 

\begin{equation}
d_g = \sum_{i=1}^{N_f} w_i + \delta N_m
\end{equation}

where $N_f$ and $N_m$ are the numbers of females and males across all patches, the first term is the summed fitnesses of all these females, and $\delta$ is a constant (range: $0-\infty$) that scales the effect of each male on $d_g$ relative to a female with fitness $w_i = 1$. This formulation means that females with high relative fitness (i.e. fecundity) have a stronger effect on the global density than do low-fitness females. I also assume that each male contributes a fixed amount to the global density, irrespective of his genotype/fitness (since I assume that male fitness only affects male mating success; see below). The parameter $\delta$ represents sex differences in ecological niche use and behaviour. For example, we might expect $\delta<1$ in species where males and females utilise very different environmental niches, or $\delta>1$ in species where males are harmful to females.

Second, I define the local density $d_j$ experienced by every female in patch $j$, as 

\begin{equation}
d_j = \sum_{i=1}^{n_{f,j}} w_i + \delta n_{m,j}
\end{equation}

where $n_{f,j}$ and $n_{m,j}$ are the numbers of females and males in patch $j$. As before, this formulation means that $d_j$ depends on the summed fitnesses of the females in the patch, as well as the number of males (scaled by the constant $\delta$).

Finally, the overall density experienced by female $i$ in patch $j$ ($D_i$) is a weighted sum of the global and local densities given by $D_i = \psi d_g + (1 - \psi)d_j$, where the parameter $\psi$ weights the importance of global and local density to female fecundity. When $\psi = 0$, only local density matters and selection on females is entirely 'soft', while when $\psi = 1$ only global density matters and selection on females is completely 'hard' (as in [@li2018ev]). Intermediate values of $\psi$ produce a mixture of hard and soft selection on females.

After calculating the expected fecundity of each female ($F_i$), we generate the realised fecundity of the female by randomly sampling from a Poisson distribution with $\lambda = F_i$ (allowing for stochastic variation in fecundity between females with equal $F_i$). If the resulting number of offspring exceeded the global carrying capacity $K$, the model randomly selects $K$ surviving offspring. 


### Competition between males

After determining how many offspring each female produces, we determine the fathers of each of these offspring. We assume that all breeding occurs within patches, such that males only compete for matings/fertilisations with males in the same patch. If the patch contains $k$ different male genotypes and there are $n_1, n_2, ... n_k$ males of each genotype, the probability that a male of genotype $k$ is the father of any given offspring is 

\begin{equation}
p_j = \frac{n_{k}w_k}{\sum_{i=1}^{k}n_{i}w_i}
\end{equation}

such that relatively common and/or high-fitness male genotypes are more likely to sire offspring. This formulation means that both sexes potentially reproduce with multiple partners.

### Reproduction, mutation and dispersal

After picking the parents, the model randomly generates each offspring's genotype according to its parents' expected gamete (and thus zygote) frequencies. Offspring are born in the same patch as their parents, and the parental generation is replaced by the offspring generation.

When an offspring is created, each $Z^+$ allele it carries has a chance $\mu_Z$ to mutate to a $Z^r$ allele, and _vice versa_ (i.e. mutation in both directions is equally probable). Similarly, each _W+_ allele has a chance $\mu_W$ to mutate to a $W^r$ allele, and _vice versa_. 

Female and male offspring disperse to another patch with probabilities $x_f$ and $x_m$ respectively. We model two types of dispersal, in separate simulations: local dispersal, in which offspring move to one of the two neighbouring patches with equal probability (recalling that the patches are arranged in a ring), or global dispersal, in which dispersing offspring can land in any of the other patches.

### One compete run of the simulation

The model first initialises a population of 10,000 individuals (the carrying capacity, $K$) with low or zero frequencies of $Z^r$, $W^r$, _A_ and _B_ alleles, higher frequencies of the wild type $Z^+$, _W+_, _a_, and _b_ alleles, and zero _Z\*_ gene drive alleles. It then runs 50 generations of burn-in to allow the population to reach demographic and genotypic equilibrium. Next, $n_{release}$ males with the genotype _Z\*Z\*aabb_ are added to the population just before fathers are selected, representing the release into the wild of a laboratory-reared strain homozygous for the driving _Z_. In some simulations, all the _Z\*Z\*aabb_ males were released in a single patch, while in others the $n_{release}$ males were randomly and evenly divided across all $k$ patches. The model continued until either A) the driving _Z\*_ allele went extinct, B) the population went extinct, C) the $W^r$ chromosome went to fixation (making population suppression impossible), D) the _Z\*_ allele fixed without causing extinction, or E) 1000 generations had elapsed. The model recorded which of these five outcomes occurred, as well as the allele frequencies, population size, and sex ratio at each generation. 

### Investigating the parameter space
For each of the parameters in Table 1, I selected two or more possible parameter values (e.g. high versus low rates of _W_-shredding $p_{shred}$; many versus few patches $k$). I then ran the model once for all possible combinations of these parameter values (n = `r n_model_runs[1]` model runs). The aim was to measure the effect of each parameter across various assumptions for the other parameters, as well as to investigate all 2-way interactions between the parameters. To gauge the relative importance of the various features of the _Z\*_ allele and the species' ecology to the extinction probability, I fit a binomial generalised linear model (GLM) with extinction as the dependent variable, and all the model parameters and their 2-way interactions as predictors. The predictors were scaled and centred before running the GLM, allowing for a meaningful ranking of the predictors by their absolute effects on extinction.


\newpage
## Supplementary tables

```{r echo=FALSE}
count_S1 <- sum(readRDS("../figures/tableS1.rds")[,2])
count_S2 <- sum(readRDS("../figures/tableS2.rds")[,2])
```


**Table S1**: The number and percentage of simulation runs (out of `r prettyNum(count_S1, big.mark = ",", scientific = FALSE)` total) that ended with the five possible outcomes, for the subset of simulation runs focusing on a _W_-shredder gene drive.
```{r, results='asis'}
make_table("../figures/tableS1.rds")
```

\vspace{0.5in}
**Table S2**: The number and percentage of simulation runs (out of `r prettyNum(count_S2, big.mark = ",", scientific = FALSE)` total) that ended with the five possible outcomes, for the subset of simulation runs focusing on a female-sterilising _Z_-linked gene drive.
```{r, results='asis'}
make_table("../figures/tableS2.rds")
```

\vspace{0.5in}
**Table S3**: List of the parameter values used to generate the simulation runs shown in Figure 2.
```{r, results='asis'}
make_table("../figures/tableS3.rds", math_mode = TRUE)
```



\newpage
# Supplementary figures

```{r fig.width = 12, fig.height = 14}
readRDS("../figures/S1_fig.rds")
```
**Figure S1**: Similar plot to Figure 3, except that the \textit{y}-axis shows the median number of generations until the \textit{W}-shredder caused extinction, among just the subset of simulations in which extinction actually occurred ($n$ = `r n_model_runs[4]` simulation runs). The median was only calculated if at least 40 simulation runs reached extinction, and the \textit{y}-axis is truncated at 150 generations.
\newpage

```{r fig.width = 15, fig.height = 15}
grid.draw(readRDS("../figures/S2_fig.rds"))
```
**Figure S2**: Heatmap illustrating the twelve strongest two-way interactions for simulations of a _W_-shredder, as determined by the effect sizes from the GLM plotted in Figure 4  ($n$ = `r n_model_runs[2]` simulation runs). The colour shows the % simulations runs that ended in extinction for each pair of parameter values.
\newpage

```{r fig.width = 10, fig.height = 10.6}
readRDS("../figures/S3_fig.rds")
```
**Figure S3**: Analagous information to Figure 3, but showing the results for a female-sterilising $Z^*$ allele instead of a \textit{W}-shredder ($n$ = `r n_model_runs[3]` simulation runs).
\newpage

```{r fig.height = 12, fig.width = 10}
readRDS("../figures/S4_fig.rds")
```
**Figure S4**: Analagous information to Figure S1, but showing the time to extinction for a female-sterilising $Z^*$ allele instead of a \textit{W}-shredder. Note that a median was only calculated if at least 40 simulation runs reached extinction ($n$ = `r n_model_runs[5]` simulation runs), and extinction only occurred when gene drive in males was strong ($p_{conv} = 0.95$).
\newpage

```{r fig.height = 8, fig.width = 8}
readRDS("../figures/S5_fig.rds")
```
**Figure S5**: Relative parameter importance in the simulations of _Z_-linked female-sterilising gene drives, for the top 25 most important main effects or two-way interactions (from a binomial GLM that included all the main effects and all their two-way interactions; $n$ = `r n_model_runs[3]` simulation runs). Each predictor variable was scaled before running the model, meaning that the absolute effect size indicates how important each parameter is to the extinction probability, given the range of values plotted in Figure S3.
\newpage

```{r fig.width = 15, fig.height = 15}
grid.draw(readRDS("../figures/S6_fig.rds"))
```
**Figure S6**: Heatmap illustrating the twelve strongest two-way interactions for simulations of a female-sterilising gene drive, as determined by the effect sizes from the GLM plotted in Figure S5 ($n$ = `r n_model_runs[3]` simulation runs). The colour shows the % simulations runs that ended in extinction for each pair of parameter values; grey squares correspond to missing estimates, which arise because of the failure to cause extinction for certain parameter combinations where $p_{conv} = 0$.



